test/fixtures/detect-existing-browser-api-object/content.js

```diff
@@ -10,12 +10,27 @@ test("browser api object in content script", (t) => {
     // On Firefox, window is not the global object for content scripts, and so we expect window.browser to not
     // be defined.
     t.equal(window.browser, undefined, "window.browser is expected to be undefined on Firefox");
+
+    try {
+      await browser.storage.sync.set({a: 'a'.repeat(10000000)});
+      t.fail('It should throw when attempting to set an object over quota');
+    } catch (error) {
+      t.ok(error instanceof Error);
```

Comment:
Is it possible to check the error message? This condition is true for almost every error, including "not defined" etc.

Reply:
Done in https://github.com/mozilla/webextension-polyfill/commit/f6aece9aea78ff4d1b9a7f9d41d6f8ef54fc5ff0